src/bp-core/admin/bp-core-admin-tools.php

```diff
@@ -1217,3 +1227,71 @@ function bp_admin_update_activity_favourite() {
 		);
     }
 }
+
+
+/**
+ * Create the invitations database table if it does not exist.
+ * Migrate outstanding group invitations if needed.
+ *
+ * @since BuddyBoss 1.3.5
+ *
+ * @return array
+ */
+function bp_admin_invitations_table() {
+	global $wpdb;
+
+	require_once( ABSPATH . 'wp-admin/includes/upgrade.php' );
+	require_once( buddypress()->plugin_dir . '/bp-core/admin/bp-core-admin-schema.php' );
+
+	/* translators: %s: the result of the action performed by the repair tool */
+	$statement = __( 'Creating the Invitations database table if it does not exist&hellip; %s', 'buddyboss' );
+	$result    = __( 'Failed to create table!', 'buddyboss' );
+
+	bp_core_install_invitations();
+
+	// Check for existence of invitations table.
+	$bp_prefix  = bp_core_get_table_prefix();
+	$table_name = "{$bp_prefix}bp_invitations";
```

Comment:
@KartikSuthar This should be a like this.

```suggestion
	$table_name = BP_Invitation_Manager::get_table_name();
```